react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -218,6 +218,8 @@ class AddPeopleDialog extends Component {
                 });
             });
         }
+
+        return true;
```

Comment:
Due to the change for this to be fire and close immediately, the state variable addToCallInProgress may be a target for removal. Also, the .thens attached to invitePeople will need to go so setState is not called on an unmounted component.